setup.py

```diff
@@ -22,7 +22,7 @@
 
 setup(
     name='gcloud',
-    version='0.8.0',
+    version='0.8.1',
```

Comment:
Please don't bump release numbers in the PR:  we will handle that when actually making a release.